src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/AdminUser.xml

```diff
@@ -16,7 +16,7 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0 https://api-platform.com/schema/metadata/resources-3.0.xsd"
 >
-    <resource class="Sylius\Component\Core\Model\AdminUser" shortName="Administrator">
+    <resource class="Sylius\Component\Core\Model\AdminUser" processor="sylius.api.state_processor.admin_user" shortName="Administrator" routePrefix="admin">
```

Comment:
If I understand correctly, this processor is only for the delete action, so maybe we should do it like [this](https://github.com/Sylius/Sylius/pull/16398#discussion_r1639521794)